Source/UnrealGT/Public/Generators/Image/GTSceneCaptureComponent2D.h

```diff
@@ -79,7 +89,11 @@ class UNREALGT_API UGTSceneCaptureComponent2D : public USceneCaptureComponent2D
     void BeginPlay() override;
 
 private:
+    const int KMaxColorIndex = 255;
```

Comment:
I would prefer no prefix (`K`) , since that's how it is handled everywhere else in the codebase.
Also consider using `constexpr` instead.

Reply:
You got it. This was force of habit (google style guide)

Reply:
No problem, I try to follow Epic's/Unreal's Code Style: https://docs.unrealengine.com/5.0/en-US/epic-cplusplus-coding-standardblueprint-debugging-in-unreal-engine/ when working on UE projects.